Production/scripts/validation_tool.py

```diff
@@ -116,40 +115,60 @@ def to_2D(histo, vbin):
   return histo.Project3D('yx').Clone()
 
 if __name__ == '__main__':
-  print ('[INFO] reading files', args.input)
+  print ('[INFO] reading files from', args.input)
   
   if args.n_threads > 1:
     ROOT.ROOT.EnableImplicitMT(args.n_threads)
 
   input_files = ROOT.std.vector('std::string')()
   
-  for file in glob.glob(args.input):
-    input_files.push_back(str(file))
+  for root, dirs, files in os.walk(args.input):
+    for ff in files:
+      if not '.root' in ff: continue
+      input_files.push_back('/'.join([root, ff]))
```

Comment:
- I suggest first to fill a list, then sort it and only after that convert to std::vector. In this way we don't need to rely on the order on which os.walk returns the files. 
- Add regex filter as an argument (with default value `.*\.root$`)

Reply:
done ed0e96117c3bac96bef3ea9ccc69e838e6f3a683